server/repository/src/db_diesel/name_row.rs

```diff
@@ -55,6 +55,7 @@ table! {
 
 allow_tables_to_appear_in_same_query!(name, item_link);
 allow_tables_to_appear_in_same_query!(name, name_link);
+allow_tables_to_appear_in_same_query!(name, encounter);
```

Comment:
Same with this one, the table have some hierachy and we should relect this in the files, i.e. move these to the files where they are needed...